cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -207,6 +207,10 @@ spec:
         - name: ROOK_ENABLE_DISCOVERY_DAEMON
           value: "true"
 
+        # Whether to start machineDisruptionBudget and machineLabel controller to watch for the osd pods and MDBs.
```

Comment:
"fencing" doesn't seem to be a term (that I can find) that Kubernetes uses in its machine disruption budget documentation. I can surmise this to be terminology we have selected for Rook uniquely, and it is a little ambiguous. We should explain this to the user better.

Also, why not call this `ROOK_ENABLE_MACHINE_DISRUPTION_BUDGET_CONTROLLER` to be more specific so we can avoid the problem of having to explain documentation in the first place?

Reply:
Machine disruption budgets are new and do apply to the fencing scenario, but for clarity agreed that this should be changed.

Reply:
Okay question: does this also enable/disable the pod disruption budget control aspect in addition to machine disruptions? I think I missed that before. If so, maybe `ROOK_ENABLE_DISRUPTION_BUDGET` would be better? It does seem to me that the intent is that the machine disruption controller is a separate thing. I just want to make sure I'm clear.